Client/Ecosia/UI/NTP/NudgeCards/NTPConfigurableNudgeCardCellViewModel.swift

```diff
@@ -13,44 +13,33 @@ protocol NTPConfigurableNudgeCardCellDelegate: AnyObject {
 
 /// ViewModel for configuring a Nudge Card Cell.
 class NTPConfigurableNudgeCardCellViewModel: HomepageViewModelProtocol {
+    var title: String {
```

Comment:
I see the point here 👌 - I guess having them with the `fatalError` was an easier update considering the state of the class. 
Wondering if we'd invest time and make it as `protocol`.

Reply:
I think there might be complexities in adding one more protocol since the `HomepageViewModelProtocol` is already used. I like the idea that this is a superclass that can be extended and has the majority of the logic, just like you initially implemented. The main reason I changed to overriding methods is because those custom contents are then much more organised as part of the subclass file itself instead of somewhat hidden in the initialisation (which in turn happens on the already big `HomepageViewModel`).

What do you think? Do you see any specific advantages to using protocol here instead?